test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -146,35 +135,13 @@ var _ = ginkgo.Describe("Downgrade [Feature:Downgrade]", func() {
 	})
 })
 
-var _ = ginkgo.Describe("etcd Upgrade [Feature:EtcdUpgrade]", func() {
```

Comment:
@spiffxp - can you please double check me on this?

I was trying to find where this tests is running and failed to find anything.

Reply:
I also couldn't find anything